Week2/alaa.js

```diff
@@ -0,0 +1,94 @@
+//1
+console.log('Hallo', 'wereld!'); // Dutch;
+console.log('hola', 'mundo!'); //spanish
+console.log('مرحبا', 'العالم!'); //Arabic
+//2
+console.log("I'm awesome"); // should use two quotations instead of one
+//3
+const x;
+console.log('the value of my variable x will be: undefined');
```

Comment:
Did you try to `console.log` it? Since `x` is a `const` it can't change after initialising, therefore `const x;` is invalid and returns a SyntaxError. The correct use here would be to use a `let` so you can change the value to `7`.